app/views/Layout.scala.html

```diff
@@ -52,12 +51,6 @@
     }
 }
 
-@beforeContentBlock = {
-    @if(showBackLink) {
-        @govukBackLink(BackLink.mimicsBrowserBackButtonViaJavaScript)
-    }
-}
-
```

Comment:
It would be better to keep this back link here rather than injecting it into every view